rules-tests/Php80/Rector/Class_/ClassPropertyAssignToConstructorPromotionRector/Fixture/skip_property_assign.php.inc

```diff
@@ -1,24 +0,0 @@
-<?php
-
-namespace Rector\Tests\Php80\Rector\Class_\ClassPropertyAssignToConstructorPromotionRector\Fixture;
-
-final class SkipPropertyAssign
-{
-    private $command;
-
-    public function __construct(\Closure $command)
-    {
-        $this->command = $command;
-    }
-
-    public function getCommand()
-    {
-        if (!$this->command instanceof \Closure) {
-            return $this->command;
-        }
-
-        $this->command = ($this->command)();
-
-        return $this->command;
-    }
```

Comment:
@TomasVotruba I removed this test fixture, because for whatever type assigned in `__construct()`, if it replaced in different method, it will always be new type, which I think very rare use case, eg on symfony:

https://github.com/symfony/symfony/blob/c82d8c99090346aaf938346a5342625e6eafc02e/src/Symfony/Component/Console/Command/LazyCommand.php#L194

The `ClassPropertyAssignToConstructorPromotionRector` don't cover that, as other type when re-assign with other type in other method will have different type as well.